books/RayTracingInOneWeekend.html

```diff
@@ -2139,37 +2252,39 @@
 True Lambertian Reflection
 ---------------------------
 <div class='together'>
-The rejection method presented here produces random points in the unit ball offset along the surface
-normal. This corresponds to picking directions on the hemisphere with high probability close to the
-normal, and a lower probability of scattering rays at grazing angles. This distribution scales by
-the $\cos^3 (\phi)$ where $\phi$ is the angle from the normal. This is useful since light arriving
-at shallow angles spreads over a larger area, and thus has a lower contribution to the final color.
+Using a diffuse model that scatters vectors evenly about the hemisphere produces a nice and soft
+diffuse model, but we can definitely do better. A more accurate representation of real diffuse
+objects is the Lambertian distribution, which has a scattering distribution that is propertional to
+$\cos (\phi)$, where $\phi$ is the angle from the normal. This means that a ray has a higher
+probability of scattering toward the normal than it does scattering toward the horizon. This
+distribution is less uniform than pure hemispherical scattering, but does a better job of modeling
+objects in the real world. We can create this distribution by picking random points on the surface
+of the unit sphere, and then offsetting them along the surface normal.
 
-However, we are interested in a Lambertian distribution, which has a distribution of $\cos (\phi)$.
-True Lambertian has the probability higher for ray scattering close to the normal, but the
-distribution is more uniform. This is achieved by picking random points on the surface of the unit
-sphere, offset along the surface normal. Picking random points on the unit sphere can be achieved by
-picking random points _in_ the unit sphere, and then normalizing those.
+At the point of intersection on a surface there is the hit point, $\mathbf{p}$, and there is the
+normal of the surface, $\mathbf{n}$. At the point of interfection, this surface can only have two
+sides. So there can only be, at most, two unit spheres tangent to any intersection point. One
+unique sphere for each side of the surface. These two unit spheres will be displaced
+from the surface by the length of their radius, which for unit spheres is exactly one. One sphere
+will be displaced in the direction of the surface's normal ($\mathbf{n}$) and one sphere will be
+displaced in the opposite direction ($\mathbf{-n}$).
```

Comment:
Perhaps add "As a result, these spheres will just touch the surface at the intersection point $\mathbf{p}$."